ocs_ci/ocs/constants.py

```diff
@@ -450,6 +450,11 @@
 DEFAULT_STORAGECLASS_RBD_THICK = f"{DEFAULT_CLUSTERNAME}-ceph-rbd-thick"
```

Comment:
Did you make sure the usage of all of these above is replaced by your new approach using storagecluster name from config + suffix? I would delete those from constants to be used, and rather have only those suffixes and combined by suffixes.